test/fixtures/users.yml

```diff
@@ -4,8 +4,12 @@
 # model remove the '{}' from the fixture names and add the columns immediately
 # below each fixture, per the syntax in the comments below
 #
-one: {}
```

Comment:
use factory bot

Reply:
@tachyons , i have removed these changes from this PR(no longer used)